apps/meteor/client/views/room/Header/ToolBox/ToolBox.tsx

```diff
@@ -35,26 +36,30 @@ const ToolBox = ({ className }: ToolBoxProps): ReactElement => {
 	const filteredActions = actions.filter((action) => !action.featured);
 	const visibleActions = isMobile ? [] : filteredActions.slice(0, 6);
 
-	const hiddenActions: Record<string, ToolboxActionConfig> = Object.fromEntries(
-		(isMobile ? actions : filteredActions.slice(6))
-			.filter((item) => !item.disabled)
-			.map((item) => {
-				hiddenActionRenderers.current = {
-					...hiddenActionRenderers.current,
-					[item.id]: item.renderOption || renderMenuOption,
-				};
-				return [
-					item.id,
-					{
-						label: { title: t(item.title), icon: item.icon },
-						action: (): void => {
-							openTabBar(item.id);
-						},
-						...item,
-					},
-				];
-			}),
-	);
+	const hiddenActions = (isMobile ? actions : filteredActions.slice(6))
+		.filter((item) => !item.disabled && item.id !== 'start-call')
```

Comment:
Are we ignoring `start-call` because it is a featured item? 
If so, shouldn't we ignore all items which are featured instead of ignoring `start-call` by hard-coding?

Reply:
Yes, you are right. Going to change to ignore all feature items